_includes/manuals/1.16/4.1.1_authentication.md

```diff
@@ -227,6 +227,14 @@ When using Active Directory, please be aware that users will be able to log in f
 
 To change this password expiry period, see [Microsoft KB906305](https://support.microsoft.com/en-us/kb/906305) for the necessary registry change.
 
+#### Brute-force protection
+
+Foreman allows only 30 failed attempts in the last 5 minutes per one IP address by default. Any subsequent login attempts are not allowed and error message is shown: "Too many tries, please try again in a few minutes." If this is triggered by accident, the silent period can be removed by deleting failed login cache entries:
+
+    find /usr/share/foreman/tmp/cache -name failed_login\*
+
+When using memcache plugin, delete the same keys in the database instead. Amount of attempts and silent period is not configurable.
```

Comment:
using *the* memcache plugin. I'm also not sure we should refer to memcache as the database (IMHO that's still PostgreSQL or MySQL). Another difficulty is that memcache itself doesn't really have a mechanism to delete keys starting with some string.

Reply:
Ok how about this.